src/components/organisms/LocalResourceDiffModal/LocalResourceDiffModal.tsx

```diff
@@ -39,6 +39,8 @@ import {getResourceKindHandler} from '@src/kindhandlers';
 
 import * as S from './styled';
 
+export type ModalTypes = 'toCluster' | 'toLocal' | null;
```

Comment:
Using an enum is better that using a type, it can be easily refactored and it improves readability

Reply:
this is really good suggestion! fixed